tests/miral/CMakeLists.txt

```diff
@@ -50,6 +50,7 @@ mir_add_wrapped_executable(miral-test-internal NOINSTALL
     ignored_requests.cpp
     focus_mode.cpp
     fd_manager.cpp
+    application_selector.cpp
```

Comment:
Shouldn't be "internal" now we're publishing `ApplicationSelector`

Reply:
As a note, `application_selector.cpp` relies on `test_window_manager_tools.h`. `test_window_Manager_tools.h` relies on `basic_window_manager.h`, which is internal to mir.

As a result, I will have to be moving `test_window_manager.h` to `/sr/tests/miral` (unless there exists a better option, but the dependency seems very important)

Reply:
Leave it then

Reply:
Okay